packages/office-ui-fabric-react/src/components/TextField/TextField.types.ts

```diff
@@ -222,11 +222,6 @@ export interface ITextFieldProps extends React.AllHTMLAttributes<HTMLInputElemen
    */
   iconClass?: string;
 
-  /**
-   * Internal ID passed to render functions.
```

Comment:
Note: this wasn't even passed to every render function, it was just being passed to onRenderLabel.